frontend/benefit/applicant/src/components/applications/forms/application/step1/companyInfo/__tests__/CompanyInfo.test.tsx

```diff
@@ -0,0 +1,29 @@
+import { RenderResult } from '@testing-library/react';
+import renderComponent from 'benefit/applicant/__tests__/utils/render-component';
+import faker from 'faker';
+import { axe } from 'jest-axe';
+import React from 'react';
+
+import CompanyInfo, { CompanyInfoProps } from '../CompanyInfo.component';
+
+describe('CompanyInfo', () => {
+  const data = {
+    name: faker.company.companyName(),
+    city: faker.address.city(),
+    postcode: faker.address.zipCode(),
+    streetAddress: faker.address.streetAddress(),
+    businessId: faker.datatype.uuid(),
+  };
+
+  const initialProps: CompanyInfoProps = { data };
+
+  const getComponent = (props: Partial<CompanyInfoProps> = {}): RenderResult =>
+    renderComponent(<CompanyInfo {...initialProps} {...props} />);
+
+  it('should render with no accessibility violations', async () => {
+    const { container } = getComponent();
+    const results = await axe(container);
+
+    expect(results).toHaveNoViolations();
+  });
```

Comment:
Plan to add more tests? :) I have used https://github.com/nock/nock to mimic the backend responses but I'm not sure if it's a good idea, it causes to test fail when running tests parallell, so some other idea to mock the backend responses is welcomed. 
Now the test run is hacked to work on kesaseteli by using jest flag `--RunInBand` to run tests sequentially. Of course it is slower then but no random failures anymore.